Gems/GraphCanvas/Code/Source/Components/SceneComponent.cpp

```diff
@@ -1286,6 +1286,10 @@ namespace GraphCanvas
     {
         GRAPH_CANVAS_PROFILE_FUNCTION();
         const SceneComponentSaveData* saveData = saveDataContainer.FindSaveDataAs<SceneComponentSaveData>();
```

Comment:
nit: don't see this get used later, so it is just for a quick check? so move it inside if condition scope?